File: Common Library/DotNet_Framework/PredictorAPI/Models/Football/FootballTeamStat.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PredictorAPI.Models.Football
{
    public class FootballTeamStat
    {
        [JsonProperty("avg_gf")]
        public double AvgGF { get; set; }

        [JsonProperty("avg_ga")]
        public double AvgGA { get; set; }

        [JsonProperty("att_trend")]
        public double AttTrend { get; set; }

        [JsonProperty("def_trend")]
        public double DefTrend { get; set; }

        [JsonProperty("rest_date")]
        public int RestDate { get; set; }
    }
}

File: MachineLearning/Poseidon_Predictor_Server/app/main/model/football/football_team_stat.py
class FootballTeamStat:
    def __init__(self):
        self.avg_gf = None
        self.avg_ga = None
        self.att_trend = None
        self.def_trend = None
        self.rest_date = None

    def set_avg_gf(self, avg_gf):
        self.avg_gf = avg_gf

    def set_avg_ga(self, avg_ga):
        self.avg_ga = avg_ga

    def set_att_trend(self, att_trend):
        self.att_trend = att_trend

    def set_def_trend(self, def_trend):
        self.def_trend = def_trend

    def set_rest_date(self, rest_date):
        self.rest_date = rest_date
